docs/tutorials/clickstream-docker.rst

```diff
@@ -55,17 +55,18 @@ are quite large and depending on your network connection may take
 
    ::
    
-                        Name                                Command               State                Ports
-        -------------------------------------------------------------------------------------------------------------------
-        ksql-clickstream-demo_control-center_1    /etc/confluent/docker/run        Up      0.0.0.0:9021->9021/tcp
-        ksql-clickstream-demo_datagen-ratings_1   bash -c echo Waiting for K ...   Up
-        ksql-clickstream-demo_elasticsearch_1     /usr/local/bin/docker-entr ...   Up      0.0.0.0:9200->9200/tcp, 9300/tcp
-        ksql-clickstream-demo_grafana_1           /run.sh                          Up      0.0.0.0:3000->3000/tcp
-        ksql-clickstream-demo_kafka-connect_1     /etc/confluent/docker/run        Up      8083/tcp, 9092/tcp
-        ksql-clickstream-demo_kafka_1             /etc/confluent/docker/run        Up      9092/tcp
-        ksql-clickstream-demo_ksql-server_1       /etc/confluent/docker/run        Up      8088/tcp
-        ksql-clickstream-demo_schema-registry_1   /etc/confluent/docker/run        Up      8081/tcp
-        ksql-clickstream-demo_zookeeper_1         /etc/confluent/docker/run        Up      2181/tcp, 2888/tcp, 3888/tcp
+                        Name                                 Command               State                Ports
+        ----------------------------------------------------------------------------------------------------------------------
+        clickstream_datagen_1_d3706de7f63f   bash -c echo Waiting for K ...   Up
```

Comment:
For your consideration: for readability, use container names in the `docker-compose.yml` file so this would read `datagen` instead of `clickstream_datagen_1_d3706de7f63f`

Reply:
👍